php/integrations/behat.rst

```diff
@@ -112,6 +136,12 @@ However, you can control this behavior with the ``disableProfiling()`` and
         }
     }
 
+.. note::
+
+    If you use the Symfony driver, since no actual requests are made, Blackfire will interpret these profiles as coming
+    from a command, not an HTTP request, so it will apply the assertions that you have defined for your commands,
+    not your requests
```

Comment:
```suggestion
.. warning::

    If you use the Symfony driver, Blackfire will interpret these profiles as
    coming from a command, not an HTTP request, since no actual requests are made.
    The assertions defined for your commands will be evaluated, and not those
    for your HTTP requests.
```

Reply:
@carlos-granados, this said, I don't like we have to end up in this situation with Symfony driver.

For Laravel Octane (swoole-based), we were in a similar situation, but for the monitoring. Using this snippet helped flag the traces as HTTP requests.
https://github.com/blackfireio/php-sdk/blob/master/src/Blackfire/Bridge/Laravel/Listeners/OctaneRequestIntrumentationStart.php#L31C8-L36C63

@romainneutron, do you think it could also work for the profiles triggered using the Symfony Driver?

Reply:
Let me try it, I will let you know if that works

Reply:
@thomasdiluccio Unfortunately it does not seem to work. I tried both calling the `setAttribute` static function as in the Laravel Octane code and calling the `setAttribute` function on the $probe object that is created when the request is started and it still understands the profiles as coming from a command instead of an HTTP request :-(

Reply:
@romainneutron Do you see a way to force the profile to be interpreted as an HTTP call?

Reply:
unfortunately no